demisto_sdk/commands/common/tools.py

```diff
@@ -3385,3 +3408,31 @@ def field_to_cli_name(field_name: str) -> str:
         field_name (str): the incident/indicator field name.
     """
     return re.sub(NON_LETTERS_OR_NUMBERS_PATTERN, "", field_name).lower()
+
+
+def get_path_in_content_repo(file_path):
+    """Returns the relative file path in the Content repo.
+
+    Args:
+        file_path (str): the absolute file path
+    """
+
+    path = str(file_path)
+    if path.startswith("Packs"):
+        return file_path
+    file_parts = path.split("Packs")
+    return f"Packs{file_parts[1]}"
+
+
+def get_pack_display_name_from_file_in_pack(file_paths: list):
+    """
+
+    Args:
+        file_path (_type_): _description_
+    """
+    display_names = set()
+    for file_path in file_paths:
+        metadata = get_pack_metadata(file_path)
+        display_names.add(metadata.get("name"))
+
+    return list(display_names)
```

Comment:
```suggestion
    return sortred(display_names)
```
for a deterministic result